src/themes/dark.scss

```diff
@@ -1,6 +1,8 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-dark {
   --wxc-meeting-background: linear-gradient(180deg, #1A1A1A 0%, #0F0F0F 100%);
   --wxc-text-color: rgba(255, 255, 255, 0.95);
+  --wxc-meeting-border: 1px solid rgba(255, 255, 255, 0.3);
```

Comment:
please put the --wxc-text-color var at the top and group the 3 meeting variables after a blank line:
```css
--wxc-text-color:

--wxc-meeting-background
--wxc-meeting-border
--wxc-meeting-box-shadow

Reply:
also, use scss colors